code/html/calculator.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width , initial-scale=1">
+  <script src="../js/rcalc-loader.js"></script>
+  <link rel="stylesheet" id="id-page-style" type="text/css" href="../css/theme-default.css">
+  <title data-i18n="calculator.title">Calculator</title>
+</head>
+
```

Comment:
Bad formatting. You don't use some formatter in atom? 
 head and body are inside html ... should be spaced

Reply:
I am using Atom Beautify... change the settings and now it does indent even body and head tags inside html